private_gpt/server/chat/chat_service.py

```diff
@@ -141,6 +147,7 @@ def stream_chat(
             system_prompt=system_prompt,
             use_context=use_context,
             context_filter=context_filter,
+            context_template=context_template,
```

Comment:
Given we are not making use of this context_template argument anywhere, I'd suggest to remove it for the moment.